extension/chrome/elements/compose-modules/compose-recipients-module.ts

```diff
@@ -903,37 +1100,47 @@ export class ComposeRecipientsModule extends ViewModule<ComposeView> {
       } else if (firstKeyInfo.revoked) {
         recipient.status = RecipientStatus.REVOKED;
-        $(el).addClass("revoked");
+        $(el).addClass('revoked');
         Xss.sanitizePrepend(el, '<img src="/img/svgs/revoked.svg" class="revoked-or-expired">');
-        $(el).attr('title', 'Does use encryption but their public key is revoked. ' +
-          'You should ask them to send you an updated public key.\n\n' +
-          this.formatPubkeysHintText(info.sortedPubkeys));
+        $(el).attr(
+          'title',
+          'Does use encryption but their public key is revoked. ' +
+            'You should ask them to send you an updated public key.\n\n' +
+            this.formatPubkeysHintText(info.sortedPubkeys)
+        );
       } else {
         recipient.status = RecipientStatus.EXPIRED;
-        $(el).addClass("expired");
+        $(el).addClass('expired');
         Xss.sanitizePrepend(el, '<img src="/img/svgs/expired-timer.svg" class="revoked-or-expired">');
-        $(el).attr('title', 'Does use encryption but their public key is expired. ' +
-          'You should ask them to send you an updated public key.\n\n' +
-          this.formatPubkeysHintText(info.sortedPubkeys));
+        $(el).attr(
+          'title',
+          'Does use encryption but their public key is expired. ' +
+            'You should ask them to send you an updated public key.\n\n' +
+            this.formatPubkeysHintText(info.sortedPubkeys)
+        );
       }
     } else {
       recipient.status = RecipientStatus.NO_PGP;
-      $(el).addClass("no_pgp");
+      $(el).addClass('no_pgp');
       if (info?.info.name) {
         recipient.name = info.info.name;
         $(el).find('.recipient-name').text(Xss.escape(info.info.name));
       }
       Xss.sanitizePrepend(el, '<img class="lock-icon" src="/img/svgs/locked-icon.svg" />');
-      $(el).attr('title', 'Could not verify their encryption setup. You can encrypt the message with a password below. Alternatively, add their pubkey.');
+      $(el).attr(
+        'title',
+        'Could not verify their encryption setup. You can encrypt the message with a password below. Alternatively, add their pubkey.'
+      );
     }
     // Replace updated recipient in addedRecipients
     const changedIndex = this.addedRecipients.findIndex(
-      (addedRecipient) => (!recipient.email || addedRecipient.email === recipient.email) && addedRecipient.id === recipient.id
+      addedRecipient =>
+        (!recipient.email || addedRecipient.email === recipient.email) && addedRecipient.id === recipient.id
     );
     if (changedIndex > -1) {
       this.addedRecipients.splice(changedIndex, 1, recipient);
     }
-    this.view.pwdOrPubkeyContainerModule.showHideContainerAndColorSendBtn(); // tslint:disable-line:no-floating-promises
+    this.view.pwdOrPubkeyContainerModule.showHideContainerAndColorSendBtn(); // eslint-disable-line @typescript-eslint/no-floating-promises
```

Comment:
Maybe should still have `.catch(Catch.reportErr);`. That may resolve the floating promise warning anyway. Can be different PR